### PR TITLE
Added Day 7 function content 

### PR DESCRIPTION
**Pull request by Sahil-Vaidya**
provided with the best quality of content with syntax & Examples

### CODE REVIEW

File: docs/day-07/function-in-cpp.md [hunk 1]
```diff
@@ -8,19 +8,370 @@ slug: function-in-cpp
 
 TASK:
```
Comment (subhadipbhowmik) [inline docs/day-07/function-in-cpp.md:9]: REMOVE THIS TEXT

File: docs/day-07/function-in-cpp.md [hunk 2]
```diff
@@ -8,19 +8,370 @@ slug: function-in-cpp
...
-8. C++ Function with Return Value
-9. C++ Function with Default Arguments
-10. C++ Function with Function Overloading
-11. C++ Function with Recursion
-12. C++ Function with Inline Function
+### 1. What is a Function in C++?
 
+A function in C++ is a block of code that performs a specific task and can be called by its name, optionally taking input parameters and returning a value. Functions help in modularizing code, making it reusable and easier to manage.
 
 IMAGE FILE:
```
Comment (subhadipbhowmik) [inline docs/day-07/function-in-cpp.md:15]: REMOVE TEXT IMAGE FILE. AND SET THE IMAGE IN BETWEEN THE DOCS

Comment (Sahil-Vaidya) [reply docs/day-07/function-in-cpp.md:6]: done

File: docs/day-07/function-in-cpp.md (hunk 1 above)
Comment (Sahil-Vaidya) [reply docs/day-07/function-in-cpp.md]: I have updated the file but This was not visible when the file was edited.

**Pull request merged**